### DISCUSSION TITLE
v1.2-experimental1

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#2824] [Announcements]
## 1.2 Experimental (1.2.220727.1-experimental1) 🧪

The 1.2 experimental release of Windows App SDK is now available! Releases in the experimental channel include features that are in the early stages of development. Experimental features may be removed from the next release, or may never be released. See [Windows App SDK release channels](https://docs.microsoft.com/windows/apps/windows-app-sdk/release-channels) for more information about the various release channels.

This release includes bug fixes from 1.1 and provides access to non-stable APIs and features for WinUI3, Input & Composition, Content, Dispatching, and Notifications. 

**Check out the [release notes](https://docs.microsoft.com/windows/apps/windows-app-sdk/experimental-channel#version-12-experimental-120-experimental1) for all of the new and updated features, limitations, and known issues.**

### Updating to this version

The 1.2.220727.1-experimental1 package is available at: https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.2.220727.1-experimental1

Additionally, for those testing deployment of unpackaged apps, download the Windows App SDK Installers and MSIX packages [here](https://aka.ms/windowsappsdk/1.2/1.2.220727.1-experimental1/Microsoft.WindowsAppRuntime.Redist.1.2.220727.1-experimental1.zip).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.2-exp1'>v1.2-experimental1</a>.</em>

**Comment by riverar** [comment]
So. Err. What's in the roadmap? I can't find an upcoming feature list. There's just a random set of changes in the release notes for 1.2-experimental1...

**Reply by gabbybilka** [reply]
Planning to announce a feature list for 1.2 at or before the next community call on the 17th. We want to be as confident as we can that we're able to deliver the features in the list that we share for the 1.2 release.

In the next couple of months, we'll be shipping 1.2 previews that will include features that will be available in the stable release.
